src/main/python/hydra/lib/runtestbase.py

```diff
@@ -6,17 +6,28 @@
 import time
 import sys
 from hydra.lib import appserver, mmapi, util
+from ConfigParser import ConfigParser
 
 l = util.createlogger('runTestBase', logging.INFO)
+from marathon.models import MarathonApp, MarathonConstraint
 # l.setLevel(logging.DEBUG)
 
 
 class RunTestBase(object):
-    def __init__(self, test_name, config):
+    def __init__(self, test_name, config = None, config_filename = None,
```

Comment:
Probably better to move to **kwargs, otherwise we will keep adding more vars as we need it.